test-api-locally.md

```diff
@@ -1,11 +1,17 @@
 # Setup local Galasa APIs for testing
 
-### Overview
+## Overview
 
 In order to test new developments on the Galasa API servlets you need to be able to run them locally.
-The below instructions will help you setup couchDB and a local API server in order to run tests against the APIs.
+The below instructions will help you setup couchDB, etcd and a local API server in order to run tests against the corresponding galasa APIs.
 
-### 1. CouchDB
+## External Applications
+
+If you want to test the behaviour as if you are on an ecosystem you will need to install and run applications external to galasa.
+
+The galasa API service will still run if none of these applications is installed. It will use the local galasa settings in the `.galasa` directory.
+
+### 1.1 CouchDB
 
 The current version of CouchDB that is used by Galasa is version 2.3.1 which can be downloaded [here](https://couchdb.apache.org/#download)
 Alternatively you can downlaod it from the [archive](http://archive.apache.org/dist/couchdb/binary/)
```

Comment:
Can we start this on a new line, render shows it as being part of the line above